### DISCUSSION TITLE
Windows Package Manager 1.9.2507

### DISCUSSION THREAD

**Discussion by ryfu-msft** [microsoft/winget-cli#4862] [Releases]
This release represents our first Windows Package Manager 1.9 release candidate build for Windows 10 (1809+), and Windows 11. 

This release includes the ability to enable experimental features unless blocked by Group Policy Objects (GPO)

## Features
* Experimental Support for Sixel 
* Support for portables with binary dependencies
* Entra ID auth support
* V2 Index
* Security Improvements
* Detailed help documentation for powershell cmdlets

## What's Changed
* Increment version to 1.9 by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4522
* Update localization pipeline to new auth type by @florelis in https://github.com/microsoft/winget-cli/pull/4532
* Fix loc pipeline service connection by @florelis in https://github.com/microsoft/winget-cli/pull/4533
* Enable Microsoft Entra Id auth for displaycatalog and use enterprise offline sku by @yao-msft in https://github.com/microsoft/winget-cli/pull/4544
* Add tests for configure mixed elevation by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4487
* Make v2 index EF enabled by default by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4553
* Trim Version When Parsing by @Trenly in https://github.com/microsoft/winget-cli/pull/4554
* Add Tests for Parsing Versions with Whitespace by @Trenly in https://github.com/microsoft/winget-cli/pull/4557
* Winget Repair - Eliminate installer type mapping for MSI/WIX and MSIX NonStore ,code refactoring & E2E Test Coverage by @Madhusudhan-MSFT in https://github.com/microsoft/winget-cli/pull/4534
* Update Pipeline Component Versions by @Trenly in https://github.com/microsoft/winget-cli/pull/4560
* Add support for extracting archives using tar.exe by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4541
* Make SxS stable by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4563
* Use digest verification APIs for MSIX when available by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4564
* Improve Support for Versions with Preambles by @Trenly in https://github.com/microsoft/winget-cli/pull/4558
* Launch COM server for elevated use via packaged path by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4567
* Fix export manifest with multiple locales by @yao-msft in https://github.com/microsoft/winget-cli/pull/4569
* Configuration history support by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4552
* Make Store package download stable by @yao-msft in https://github.com/microsoft/winget-cli/pull/4570
* Update expired certificate with incoming one by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4568
* Apply latest patch for localized strings by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4578
* Update winget server com security by @yao-msft in https://github.com/microsoft/winget-cli/pull/4577
* Component Governance fix: Update libyaml by @yao-msft in https://github.com/microsoft/winget-cli/pull/4583
* Update valijson to v1.0.2 by @yao-msft in https://github.com/microsoft/winget-cli/pull/4588
* Use atomic functions to ensure single initialization of tracking catalog by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4592
* Standardize line endings in expect.txt by @Trenly in https://github.com/microsoft/winget-cli/pull/4595
* Fix service connection for ESRP codesigning for utils nuget package by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4597
* Configuration apply queueing by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4590
* Ensure Agreements don't terminate in dependency flow by @Trenly in https://github.com/microsoft/winget-cli/pull/4594
* Remove libyaml Dockerfile by @florelis in https://github.com/microsoft/winget-cli/pull/4604
* Update README.md and Pull Request and Issue templates by @skanda890 in https://github.com/microsoft/winget-cli/pull/4599
* Attempt to update dependencies when not registered by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4606
* Remove experimental from index v2 by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4607
* ARM64 build and test fixes by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4608
* Use shared manifest in COM server by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4605
* Add AnyCPU to .gitignore by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4621
* Fix for attempted use of capability API before it was introduced by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4620
* Fix Component Governance by @msftrubengu in https://github.com/microsoft/winget-cli/pull/4629
* Update documentation errors by @Trenly in https://github.com/microsoft/winget-cli/pull/4649
* Update Zlib library dependency for PureLib by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4654
* update curl vcpkg version to 8.8.0 by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4655
* Pull in latest SFSClient by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4670
* Remove prerelease, update icon uri, and clean up for PowerShell modules by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4672
* Use ContinueWith rather than finally for coroutine result types by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4669
* Store configuration status and enable remote watch events by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4647
* Update README.md by @skanda890 in https://github.com/microsoft/winget-cli/pull/4687
* Update metadata collection tooling by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4688
* Skip uninstall attempt when installed package version is not present by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4685
* Upgrade to check-spelling v0.0.22 by @jsoref in https://github.com/microsoft/winget-cli/pull/3896
* Fix downloader selection by @florelis in https://github.com/microsoft/winget-cli/pull/4696
* Split testing into separate pipeline job by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4675
* Update NOTICE by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4699
* Delete CodeCoverage and WinGetUtil NuGet pipelines by @florelis in https://github.com/microsoft/winget-cli/pull/4708
* WinRT activation shim for Microsoft.Management.Deployment by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4709
* Add missing solution dependency by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4717
* Fix the actual problem with the build... by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4719
* Send unit telemetry on success by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4720
* Pull in latest SFS client by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4725
* PowerShell Help for Microsoft.WinGet.Client by @denelon in https://github.com/microsoft/winget-cli/pull/4674
* Create public sample by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4734
* Generate Microsoft.WinGet.Client help docs in cmdlet output by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4731
* Remove preference for "installed" installer type by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4740
* Update curl to 8.9.1 by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4745
* Fix Reset-WinGetSource behavior by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4732
* External HR error message support by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4746
* Generate test certificates in pipeline by @florelis in https://github.com/microsoft/winget-cli/pull/4747
* [Spec] Windows Package Manager Repair functionality Design spec by @Madhusudhan-MSFT in https://github.com/microsoft/winget-cli/pull/3943
* Update build status badge by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4758
* Run test tasks if succeededOrFailed by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4759
* Binskim error EnableCriticalCompilerWarnings by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4764
* Run Unpackaged Unit Tests with winget-installed PsExec by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4760
* Spelling by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4772
* Updates to PowerShell help docs by @mdanish-kh in https://github.com/microsoft/winget-cli/pull/4781
* Repair-WinGetPackageManager improvements by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4774
* Allow for smaller configuration outputs by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4754
* Add diagnostics to dynamic factory by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4757
* PS Gallery tests are unreliable by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4791
* Add pwsh support for trust level and explicit by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4750
* Updates to the PowerShell modules by @kilasuit in https://github.com/microsoft/winget-cli/pull/4716
* Microsoft.WinGet.DSC improvements (and breaking change) by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4795
* Small build changes by @florelis in https://github.com/microsoft/winget-cli/pull/4807
* Add initial 1.9 manifest by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4811
* Fix Component Governance issues by @florelis in https://github.com/microsoft/winget-cli/pull/4814
* Fix grammatical issues in Bug Report.yml, and Update dependencies by @skanda890 in https://github.com/microsoft/winget-cli/pull/4788
* upgrading to similar-issues-ai by @denelon in https://github.com/microsoft/winget-cli/pull/4819
* Update nuget for E2E test project  by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4821
* Fix PATH issue for archive containing nested portables with binary dependencies by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4816
* Revert PowerShell modules to alpha by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4823
* Update fuzzing template area path by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4824
* Add a few more command aliases by @Trenly in https://github.com/microsoft/winget-cli/pull/4822
* Fix issue with unicode characters in symlink target path by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4834
* Pull in latest localized strings by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4840
* Sixel support by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4828
* More messages for common Store package download scenarios by @yao-msft in https://github.com/microsoft/winget-cli/pull/4841
* WinGet COM API for Repair by @Madhusudhan-MSFT in https://github.com/microsoft/winget-cli/pull/4736
* Improve configuration self elevation flow by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4844
* Pull in latest localized strings by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4855
* Update curl by @florelis in https://github.com/microsoft/winget-cli/pull/4856
* Add extended error codes to results without them by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4858

## New Contributors
* @skanda890 made their first contribution in https://github.com/microsoft/winget-cli/pull/4599
* @kilasuit made their first contribution in https://github.com/microsoft/winget-cli/pull/4716

**Full Changelog**: https://github.com/microsoft/winget-cli/compare/v1.8.1911...v1.9.2507

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-cli/releases/tag/v1.9.2507'>Windows Package Manager 1.9.2507</a>.</em>

**Comment by felipebraga** [comment]
It seems **V2 Index** breaks previous version

**Reply by denelon** [reply]
Can you describe the scenario?
